tests/test_tilted_elements.py

```diff
@@ -0,0 +1,51 @@
+import torch
```

Comment:
@cr-xu Is there a reason you didn't just put these into `test_quadrupole.py`? That seems more logical to me.